src/components/Contentful/Hero.js

```diff
@@ -33,16 +42,17 @@ export default function ContentfulHero({
   return (
     <div className="contentful-hero" id={id}>
       <div className="container">
-        <div className="row">
-          <div className="col-xl-12 col-lg-12 col-md-12 col-sm-12 col-12">
+        <div className="row contentful-hero__row">
+          <div className="col-xl-6 col-lg-6 col-md-12 col-sm-12 col-12 contentful-hero__text-box">
```

Comment:
Same here, prefer child or parent divs rather than combining bootstrap and custom components.